src/c_wrapper.jl

```diff
@@ -74,6 +74,7 @@ end
 mutable struct ScsInfo{T} <: AbstractSCSType
     iter::T
     status::NTuple{128,Cchar}
+    lin_sys_solver::NTuple{128,Cchar}
```

Comment:
:( This isn't `3.2` then! That's a breaking change...

Reply:
only if `ScsInfo` is part of the official interface. Since we don't export it I don't consider it as such.

Reply:
I meant in the underlying SCS library

Reply:
ok, then I misunderstood :)